pkg/operator/ceph/controller/spec.go

```diff
@@ -74,23 +76,40 @@ var logger = capnslog.NewPackageLogger("github.com/rook/rook", "ceph-spec")
 var (
 	cronLogRotate = `
 CEPH_CLIENT_ID=%s
-PERIODICITY=%s
-LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
+PERIODICITY_IN_HOUR=%s
+LOG_ROTATE_CEPH_FOLDER=/etc/logrotate.d/ceph
+DAEMON_LOG_FILE=/var/log/ceph/$CEPH_CLIENT_ID.log
+LOG_MAX_SIZE=%s
 
-if [ -z "$PERIODICITY" ]; then
-	PERIODICITY=24h
-fi
+LAST_ROTATION_TIME_IN_SEC=$(echo $SECONDS)
 
 # edit the logrotate file to only rotate a specific daemon log
 # otherwise we will logrotate log files without reloading certain daemons
 # this might happen when multiple daemons run on the same machine
-sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FILE"
+sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FOLDER"
```

Comment:
I thought we decided everything was based on individual daemon filenames now
```suggestion
sed -i "s|*.log|$CEPH_CLIENT_ID.log|" "$LOG_ROTATE_CEPH_FILE"
```

Reply:
yes, but above is the logrotate config file, not the actual log life

Reply:
I see, it was already using the folder, and the var was just renamed